src/rules/at-function-named-arguments/__tests__/index.js

```diff
@@ -0,0 +1,852 @@
+    },
+    {
+      code: `
+      .b {
+        border: reset($value: 40px, 10px, 'black');
+      }
+    `,
+      line: 3,
+      column: 9,
+      message: messages.expected,
+      description:
+        "Always. Example: first argument is named but remaining are not."
+    },
+    {
+      code: `
+      .b {
+        border: reset(40px, $value: 10px, 'black');
+      }
+    `,
+      line: 3,
+      column: 9,
+      message: messages.expected,
+      description: "Always. Example: mixed named arguments."
+    }
+  ]
+});
+
+// Not allowed ("never")
+testRule(rule, {
+  ruleName,
+  config: ["never"],
+  syntax: "scss",
+
+  accept: [
+    {
+      code: `
+      .b {
+        background-image: linear-gradient(0deg, blue, green 40%, red);
+      }
+      `,
+      description: "Never. Example: native CSS function is ignored."
+    },
+    {
+      code: `
+      .b {
+        transform: translateX(40px) translatey(10px);
+      }
+      `,
+      description:
+        "Never. Example: native CSS camel-casing function is ignored."
+    },
+    {
+      code: `
+      .b {
+        border: reset();
+      }
+      `,
+      description: "Never. Example: no arguments with parenthesis."
+    },
+    {
+      code: `
+      .b {
+        border: reset(40px);
+      }
+    `,
+      description: "Never. Example: single argument that is not named."
+    },
+    {
+      code: `
+      .b {
+        border: reset(40px, 10px);
+      }
+    `,
+      description: "Never. Example: multiple arguments that are not named."
+    },
+    {
+      code: `
+      .b {
+        border: reset(
+          40px,
+          10px
+        );
+      }
+    `,
+      description:
+        "Never. Example: multiple arguments that are not named in multiline function call."
+    },
+    {
+      code: `
+      .b {
+        border: reset($duration);
+      }
+    `,
+      description:
+        "Never. Example: single argument is a variable and are not named."
+    },
+    {
+      code: `
+      .b {
+        border: reset(30 * 25ms);
+      }
+    `,
+      description:
+        "Never. Example: single argument is a calculated value and not named."
+    },
+    {
+      code: `
+      .b {
+        border: reset('black');
+      }
+    `,
+      description:
+        "Never. Example: single argument is a quoted string and not named."
+    },
+    {
+      code: `
+      .b {
+        border: reset(black);
+      }
+    `,
+      description:
+        "Never. Example: single argument is an unquoted string and not named."
+    },
+    {
+      code: `
+      .b {
+        border: reset(#{$value});
+      }
+    `,
+      description:
+        "Never. Example: single argument is an interpolated value and not named."
+    }
+  ],
+
+  reject: [
+    {
+      code: `
+      .b {
+        border: reset($value: 40px);
+      }
+      `,
+      line: 3,
+      column: 9,
+      message: messages.unexpected,
```

Comment:
Here and in the lines below the message should be `messages.rejected` as there is no `messages.unexpected` defined.

This also happens in the the tests for the other rule.